### ISSUE TITLE
Script Install Does not Work & Breaks things

### ISSUE THREAD

**Issue by drcrook1** [Microsoft/CNTK#1059]
Hey All,

The script install does not work and in fact it breaks my keras/theano install.  I have to disable cnmem in theano.rc for it to work again, I have not tested with larger data; I had 90% of GPU dedicated.

**My Box:**
Alienware R2 w/Graphics Amplifier NVidia 1080
Windows 10
Cuda 8.0 + CuDNN 5.1

I installed Keras + Theano for GPU via these instructions: http://ankivil.com/installing-keras-theano-and-dependencies-on-windows-10/ 

I was able to verify my cuda install works by running the Oceans sample in VS 2013 Update 5.

I followed these instructions to install CNTK: https://github.com/Microsoft/CNTK/wiki/Setup-Windows-Binary-Script and used this release: CNTK for Windows v.2.0 Beta 3 GPU 

Based on the error; it appears that CNTK has linked to Cuda 7.5, while I have configured my machine to use Cuda 8.0.  Perhaps the solution is to point CNTK to Cuda 8.0; if this is the case, how does one do this?

This is the error I receive:

(C:\local\Anaconda3-4.1.1-Windows-x86_64\envs\cntk-py34) c:\repos\CNTK\Examples\Other\Simple2d\Data>cntk configFile=../Config/Simple.cntk makeMode=false command=Simple_Demo_Train deviceId=0
CNTK 2.0.beta3.0 (HEAD d7c909, Nov 10 2016 01:50:21) on DESKTOP-4CLB1VA at 2016/11/17 17:14:11

cntk  configFile=../Config/Simple.cntk  makeMode=false  command=Simple_Demo_Train  deviceId=0
-------------------------------------------------------------------
Build info:

                Built time: Nov 10 2016 01:50:21
                Last modified date: Thu Nov 10 01:26:34 2016
                Build type: Release
                Build target: GPU
                With 1bit-SGD: no
                Math lib: mkl
                CUDA_PATH: C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v7.5
                CUB_PATH: c:\src\cub-1.4.1
                CUDNN_PATH: c:\NVIDIA\cudnn-5.1\cuda
                Build Branch: HEAD
                Build SHA1: d7c90988d36a569b0ef561e868e7c9672e245a60
                Built by svcphil on dphaim-26-hv1
                Build Path: C:\jenkins\workspace\CNTK-Build-Windows\Source\CNTK\
-------------------------------------------------------------------
-------------------------------------------------------------------
GPU info:

                Device[0]: cores = 3840; computeCapability = 6.1; type = "GeForce GTX 1080"; memory = 8192 MB
-------------------------------------------------------------------

Configuration After Processing and Variable Resolution:

configparameters: Simple.cntk:command=Simple_Demo_Train
configparameters: Simple.cntk:ConfigDir=../Config
configparameters: Simple.cntk:DataDir=../Data
configparameters: Simple.cntk:deviceId=0
configparameters: Simple.cntk:makeMode=false
configparameters: Simple.cntk:ModelDir=../Output/Models
configparameters: Simple.cntk:modelPath=../Output/Models/simple.dnn
configparameters: Simple.cntk:OutputDir=../Output
configparameters: Simple.cntk:outputNodeNames=ScaledLogLikelihood
configparameters: Simple.cntk:precision=float
configparameters: Simple.cntk:RootDir=..
configparameters: Simple.cntk:Simple_Demo_Output=[
    action = "write"
    reader = [
        readerType = "CNTKTextFormatReader"
        file = "../Data/SimpleDataTest_cntk_text.txt"
        input = [
            features = [
dim = 2
                format = "dense"
            ]
            labels = [
dim = 2
                format = "dense"
            ]
        ]
    ]
outputNodeNames = PosteriorProb : labels
outputPath = "../Output/SimpleOutput"
    format = [
type = "category"
labelMappingFile = "../Data/SimpleMapping.txt"
sequenceEpilogue = "\t// %s\n"
    ]
]

configparameters: Simple.cntk:Simple_Demo_Test=[
    action = "test"
    reader = [
        readerType = "CNTKTextFormatReader"
        file = "../Data/SimpleDataTest_cntk_text.txt"
        input = [
            features = [
dim = 2
                format = "dense"
            ]
            labels = [
dim = 2
                format = "dense"
            ]
        ]
    ]
]

configparameters: Simple.cntk:Simple_Demo_Train=[
    action = "train"
    SimpleNetworkBuilder = [
        layerSizes = 2:50*2:2
        trainingCriterion = "CrossEntropyWithSoftmax"
        evalCriterion = "ClassificationError"
        layerTypes = "Sigmoid"
        initValueScale = 1.0
        applyMeanVarNorm = true
        uniformInit = true
        needPrior = true
    ]
    SGD = [
        epochSize = 0
        minibatchSize = 25
        learningRatesPerMB = 0.5:0.2*20:0.1
        momentumPerMB = 0.9
        dropoutRate = 0.0
        maxEpochs = 10
    ]
    reader = [
        readerType = "CNTKTextFormatReader"
        file = "../Data/SimpleDataTrain_cntk_text.txt"
        input = [
            features = [
dim = 2
                format = "dense"
            ]
            labels = [
dim = 2
                format = "dense"
            ]
        ]
    ]
]

configparameters: Simple.cntk:traceLevel=1
Commands: Simple_Demo_Train
precision = "float"

##############################################################################
#                                                                            #
# Simple_Demo_Train command (train action)                                   #
#                                                                            #
##############################################################################

Creating virgin network.
SimpleNetworkBuilder Using GPU 0
Microsoft::MSR::CNTK::GPUMatrix<ElemType>::SetUniformRandomValue (GPU): creating curand object with seed 1, sizeof(ElemType)==4
CURAND failure 201: (see curand.h & look for curandStatus or CURAND_STATUS_xxx) ; GPU=0 ; hostname=DESKTOP-4CLB1VA ; expr=curandGenerateUniform(((curandGenerator_t*) s_curandGenerator)[0], reinterpret_cast<float*>(Data()), GetNumElements())

[CALL STACK]
    > Microsoft::MSR::CNTK::GPUMatrix<float>::  SetUniformRandomValue
    - Microsoft::MSR::CNTK::Matrix<float>::  SetUniformRandomValue
    - Microsoft::MSR::CNTK::TracingGPUMemoryAllocator::  operator=
    - Microsoft::MSR::CNTK::TracingGPUMemoryAllocator::  operator=
    - Microsoft::MSR::CNTK::TracingGPUMemoryAllocator::  operator=
    - Microsoft::MSR::CNTK::TracingGPUMemoryAllocator::  operator=
    - Microsoft::MSR::CNTK::TracingGPUMemoryAllocator::  operator=
    - Microsoft::MSR::CNTK::TracingGPUMemoryAllocator::  operator=
    - Microsoft::MSR::CNTK::TracingGPUMemoryAllocator::  operator=
    - Microsoft::MSR::CNTK::TracingGPUMemoryAllocator::  operator=
    - Microsoft::MSR::CNTK::TracingGPUMemoryAllocator::  operator=
    - Microsoft::MSR::CNTK::TracingGPUMemoryAllocator::  operator=
    - Microsoft::MSR::CNTK::TracingGPUMemoryAllocator::  operator=
    - Microsoft::MSR::CNTK::TracingGPUMemoryAllocator::  operator=
    - Microsoft::MSR::CNTK::TracingGPUMemoryAllocator::  operator=
    - Microsoft::MSR::CNTK::TracingGPUMemoryAllocator::  operator=

EXCEPTION occurred: CURAND failure 201: (see curand.h & look for curandStatus or CURAND_STATUS_xxx) ; GPU=0 ; hostname=DESKTOP-4CLB1VA ; expr=curandGenerateUniform(((curandGenerator_t*) s_curandGenerator)[0], reinterpret_cast<float*>(Data()), GetNumElements())

**Comment by wolfma61** [comment]
Hello - 
sorry for the trouble - yes we are still using cuda7.5 in the binary drop, and you will get the curand error with your Pascal card.

We will have a prepared cuda8 build, probably next week, that will fix the issue.
Currently the only way around is do a CUDA8 build from source yourself. Following the build - preparation step, adding CUDA8 and a cudnn version for Cuda8, and then do a build with VS2013.5 should work.

thx
Wolfgang

**Comment by wolfma61** [comment]
no follow-up for a long time, please open a new issue if you experience this in a more recent CNTK installation
